### ISSUE TITLE
Add book genre field

### ISSUE THREAD

**Issue by FinixFighter** [fenimore/badreads#29]
It would be nice to have a "Genre" field, for specifying the book genre. It should be a multiple value, since for example we could have a book which is a "Fiction","Thriller" and "Mystery". 
Then, it would be quite easy to implement a filter to show books of a specific genre (or also additional kinds of statistics etc..)

**Comment by fenimore** [comment]
Good idea, i think the open library API probably has some of this info too

**Comment by FinixFighter** [comment]
> Good idea, i think the open library API probably has some of this info too

Yes, it calls it "subjects". Theare are usually a lot of subjects for a book on OpenLibrary, but users can always edit them and leave just the preferred ones.
